library/rocksdbjni/BuilderLinux.kt

```diff
@@ -7,7 +7,7 @@ fun main() {
         val baseDir = Paths.get(".")
         val version = Paths.get("library").resolve("rocksdbjni").resolve("VERSION").toFile().useLines { it.firstOrNull() }
 
-        val javaHome = Paths.get("/opt/java-se-8u41-ri")
+        val javaHome = Paths.get("/usr/lib/jvm/java-11-openjdk-amd64")
```

Comment:
Use Java 11 JDK instead of Java. This path corresponds to Ubuntu installation directory of JDK 11, which was tested on GCP.